isodatetime/dumpers.py

```diff
@@ -226,7 +226,7 @@ def get_time_zone(self, time_zone_string):
             return None
         info = self._timepoint_parser.process_time_zone_info(info)
         if info.get('time_zone_utc'):
-            return (0, 0)
+            return 0, 0
```

Comment:
Out of interest is this safety for Python3 or preference?

Reply:
Oh, not necessary for Python 3. It was just among the warnings raised by my IDE. As I went through the warnings I tried to stick with the ones that were related to Py2 to Py3, but this one slipped in. I can remove these changes from the pull request if that's easier to review.